mozart-fetcher.spec

```diff
@@ -50,8 +50,7 @@ systemctl enable cloudformation-signal
 /bin/chown -R component:component /var/log/component
 
 %files
-/home/component
-%attr(0755, component, component) /home/component/component-status-cfn-signal.sh
+%attr(0755, component, component) /home/component/*
```

Comment:
so you are resolving the warning by changing the permissions octal value?

Reply:
Not exactly, I don't think. I think what solves the warning is removing `/home/component` from line 53, which lists all files under that folder, including `component-status-cfn-signal.sh`. So, because that file is the only file in the folder, the octal value is the same as it was before.